curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/use-bracket-notation-to-find-the-nth-to-last-character-in-a-string.english.md

```diff
@@ -38,15 +38,11 @@ tests:
 <div id='js-seed'>
 
 ```js
-// Example
-var firstName = "Ada";
-var thirdToLastLetterOfFirstName = firstName[firstName.length - 3];
```

Comment:
Hint again.

Reply:
Good catch. I will try to move the example into the description section.

Reply:
I added an example based on the challenge description.